### PR TITLE
Add changes for MinIO RELEASE.2023-03-20T20-16-18Z.

### LINKED ISSUE #767: [RELEASE] MinIO  RELEASE.2023-03-20T20-16-18Z
MinIO [RELEASE.2023-03-20T20-16-18Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-03-20T20-16-18Z) has changes that impact the docs:

- [ ] Pools with tiered content can now be decommissioned ([PR #16751](https://github.com/minio/minio/pull/16751))
- [ ] LDAP IDP can now be enabled or disabled with a flag ([PR #16805](https://github.com/minio/minio/pull/16805))
- [ ] Adds support for batch job cancellation (requires mc [RELEASE.2023-03-20T17-17-53Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-03-20T17-17-53Z)) 
see related docs issue #766 
- [ ] Object names on Windows cannot have a `\` character ([PR #16856](https://github.com/minio/minio/pull/16856))

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #767

- Adds behavior note about decommissioning pools with tiering enabled
- Adds info about enabling/disabling LDAP IDP connection
- Adds a Windows-only note in the Object Management about not using the \ character

Staged:
- http://192.241.195.202:9000/staging/minio-2023-03-20/index.html
- Windows at: http://192.241.195.202:9000/staging/minio-2023-03-20/windows/administration/object-management.html#object-organization-and-planning

### CODE REVIEW

File: source/operations/install-deploy-manage/decommission-server-pool.rst [hunk 1]
```diff
@@ -181,6 +181,19 @@ If the decommission fails, customers should open a |SUBNET| issue for further as
...
+
+.. versionchanged:: RELEASE.2023-03-20T20-16-18Z
+
+For deployments with tiering enabled and active in one or more buckets, decommissioning proceeds as expected.
+Objects subject to tiering rules move to an active pool as expected.
+
+.. note:: 
+
+   For older versions, active tiering prevents decommissioning from starting. 
+   Upgrade to RELEASE.2023-03-20T20-16-18Z or later to decommission pools where tiering is active.
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/decommission-server-pool.rst:195]: As expected leverages an assumption of how this should work.

Spitballing

```suggestion
.. versionchanged:: RELEASE.2023-03-20T20-16-18Z

   For deployments with tiering enabled and active, decommissioning moves the object references to a new active pool.
   Applications can continue issuing GET requests against those objects where MinIO handles transparently retrieving them from the remote tier.

In older MinIO versions, tiering configurations prevent decommissioning. 
You must upgrade to enable decommissioning with tiering configurations
```

Not sure the last sentence is necessary. Also don't need the note.

**Review by ravindk89 — CHANGES_REQUESTED** [review]
One area to tighten, but otherwise good.

File: source/operations/install-deploy-manage/decommission-server-pool.rst (hunk 1 above)
Comment (feorlen) [reply source/operations/install-deploy-manage/decommission-server-pool.rst:195]: Concur. As someone just getting started with this, the additional context is the difference between "does this mean what I think it does?" and "oh right I know what this is about." Yay.

**Review by feorlen — APPROVED** [review]

**Review by feorlen — APPROVED** [review]
LGTM, agree about Ravind's suggestion.

File: source/operations/install-deploy-manage/decommission-server-pool.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/decommission-server-pool.rst]: I like it. Done!

**Pull request merged**